java/arcs/sdk/wasm/WasmCombinedSingletonHandle.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.sdk.wasm
```

Comment:
Is there a reason why this stuff is going in the wasm variant?

Reply:
The Wasm stuff is forked from the JVM stuff temporarily, and most of the sample tutorial particles only compile/run in the Wasm/DevEx environment. This allows Sarah to move quickly and not block on us, or us block on her, until ProdEx can run her recipes.

Reply:
^Ditto.